doc/getting-started/getting-started.rst

```diff
@@ -710,7 +710,13 @@ Launch the User VM
       acrn@vecow:~$ uname -r
       5.10.78-acrn-service-vm
 
-The User VM has launched successfully. You have completed this ACRN setup.
+   The User VM has launched successfully. You have completed this ACRN setup.
+
+#. (Optional) To exit the User VM, run this command:
```

Comment:
How about we make it more explicit that this should be done from the terminal that's connected to the User VM? We talk about two different terminals in the step just before and people _could_ issue that command in the Service VM terminal, which would shut down the entire system here ;-)